mypy/build.py

```diff
@@ -319,7 +325,11 @@ def import_priority(imp: ImportBase, toplevel_priority: int) -> int:
     return toplevel_priority
 
 
-def load_plugins(options: Options, errors: Errors) -> Tuple[Plugin, Dict[str, str]]:
+def load_plugins(
+        options: Options,
+        errors: Errors,
+        stdout: TextIO = sys.stdout
```

Comment:
Same comment as above -- or perhaps it's possible to make this a mandatory parameter?

Also, add a trailing comma.